source/android/adaptivecards/src/main/java/io/adaptivecards/renderer/action/ActionSetRenderer.java

```diff
@@ -54,7 +54,7 @@ public View render(
         LinearLayout rootLayout = new LinearLayout(context);
         // Add tag for rendering process
         rootLayout.setTag(new TagContent(actionSet));
-        rootLayout.setLayoutParams(new LinearLayout.LayoutParams(ViewGroup.LayoutParams.MATCH_PARENT, ViewGroup.LayoutParams.MATCH_PARENT));
+        rootLayout.setLayoutParams(new LinearLayout.LayoutParams(ViewGroup.LayoutParams.MATCH_PARENT, ViewGroup.LayoutParams.WRAP_CONTENT));
```

Comment:
The changes in this file are not related to this issue

Reply:
@almedina-ms 

As per my PR description above - it was to address a regression that occurred when I bump the SDK/support library to 28. ActionSet rendering broke. Android SDK may have made changes to its layout behavior, I'm not exactly sure.